control/flags.go

```diff
@@ -19,19 +19,21 @@ limitations under the License.
 
 package control
 
-import "github.com/codegangsta/cli"
+import (
+	"fmt"
+
+	"github.com/codegangsta/cli"
+)
 
 var (
-	flNumberOfPLs = cli.IntFlag{
+	flNumberOfPLs = cli.StringFlag{
```

Comment:
I'm not sure I agree with making everything a stringflag. 

The problem it's trying to solve (I think) is that ctx.isSet will not return true for environment variables? Would it be easier/cleaner to just check if the env variable is set ourselves where we check ctx.IsSet?